### ISSUE TITLE
Unable to Reproduce Reported Results for Olmo3-7B-think on Medical task

### ISSUE THREAD

**Issue by yufeng-04** [idanshen/Self-Distillation#15]
I am currently attempting to reproduce the evaluation results for Olmo3-7B-think on the Medical task. Using the code provided in the repo and the training set mentioned in the paper, I have already tried various hyperparameter configurations, but the final accuracy is consistently much lower (at least 10% lower) than the numbers reported in the paper.

The evaluation pipeline I used was constructed strictly following the procedure mentioned in the paper, and it produces similar accuracy for the original Olmo3-7B-think.

Could you please provide more details on the experimental setup for this part of your experiments? Specifically, I would like to know the max_completion_length, epochs, learning_rate, batch_size, and ref_model_mixup_alpha used to produce the results reported in the paper, and whether mask_truncated_completions was set to True in the training process.
